WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteViewModel.kt

```diff
@@ -265,8 +265,8 @@ class MySiteViewModel @Inject constructor(
             result.filter { it.siteId == null || it.state.site != null }.mapSafe { it.state }
         }
 
-    val uiModel: LiveData<UiModel> = state.map {
-        with(it) {
+    val uiModel: LiveData<UiModel> = merge(state, quickLinksItemViewModelSlice.uiState) { cards, quickLinks ->
+        with(requireNotNull(cards)) {
```

Comment:
@AjeshRPai  - is using `requireNotNull` exposing us to a potential app crash or will there ever be a situation in which there are no cards?

Reply:
No cards scenario doesnt exists AFAIK. Since we are showing personalize card + No cards message.

Reply:
Excellent; thanks for the sanity check. 👍